postgres/10.0/Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM postgres:10.0
 MAINTAINER Debezium Community
 ARG USE_POSTGIS=true
 ENV PLUGIN_VERSION=v0.7.5
-ENV WAL2JSON_COMMIT_ID=f68cb0096c669a2ee5a2d32b54a535513e3cb23b
+
+# https://github.com/eulerto/wal2json/releases/tag/wal2json_1_0
+ENV WAL2JSON_COMMIT_ID=d2b7fef021c46e0d429f2c1768de361069e58696
```

Comment:
If there is a tag why do you refere to it vai commit id and not tag name?

Reply:
I thought we might go back to a non-tagged commit some time soon and also I wasn't sure whether I'd have to change more things then, i.e. does it work with the tag in the same way as with a commit?